src/securityPolicy.c

```diff
@@ -162,6 +181,8 @@ security_policy_t policyForGetExtendedPublicKeyBulkExport(const bip44_path_t* pa
 	case PATH_MULTISIG_SPENDING_KEY:
 	case PATH_MULTISIG_STAKING_KEY:
 	case PATH_MINT_KEY:
+	case PATH_GOVERNANCE_VOTING_ACCOUNT:
+	case PATH_GOVERNANCE_VOTING_KEY:
```

Comment:
Unrelated to this specific line, but maybe we could add a governance key case also to `ui_displayGetPublicKeyPathScreen`?

Reply:
Possibly, but it would not fit in a single screen, so I'd rather avoid that if no one is asking.